testgrid/specs/latest.yaml

```diff
@@ -0,0 +1,20 @@
+- name: ""
```

Comment:
can we also include the spec for longhorn? https://kurl.sh/longhorn

Reply:
I'll add